requirements.txt

```diff
@@ -0,0 +1 @@
+git+git://github.com/david415/python-ftools.git#egg=ftools
```

Comment:
I dont consider vendoring ftools an option since it's compiled c and this
is an otherwise pure-python project. Perhaps start by asking david415 if he
can publish it?

On Tue, Apr 5, 2016 at 9:57 AM, Brian Hatfield <notifications@github.com>
wrote:

> In requirements.txt
> <https://github.com/graphite-project/whisper/pull/162#discussion_r58553201>
> :
>
> > @@ -0,0 +1 @@
> > +git+git://github.com/david415/python-ftools.git#egg=ftools
>
> Should we add it/a fork to pypi? Is it simple enough to vendor? @mleinart
> <https://github.com/mleinart> mentioned an alternative approach - would
> that make more sense? I think some action to make this dependency be
> available in a less likely to blow up way is warranted here.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/graphite-project/whisper/pull/162/files/007c66bce649735fb474d23d392956c6e2070c6d#r58553201>
>